pkg/operation/botanist/health_check.go

```diff
@@ -470,13 +470,13 @@ func (b *HealthChecker) CheckMonitoringSystemComponents(
 
 // CheckMonitoringControlPlane checks whether the monitoring in the given listers are complete and healthy.
 func (b *HealthChecker) CheckMonitoringControlPlane(
-	namespace string,
+	botanist *Botanist,
```

Comment:
The botanist is a very heavy struct that you should only pass if it's really necessary - How about keeping `namespace` and add `gardenv1beta1.Shoot` to the arguments since the shoot is the only additional information you need?